pkg/textparse/lex.l

```diff
@@ -65,65 +54,31 @@ M     [a-zA-Z_:]
 
 %%
 
-\0                                    return eof
-#[^\r\n]*\n                           l.mstart = l.i
-[\r\n \t]+                            l.mstart = l.i
-
-{M}({M}|{D})*                         l.state = lstateName
-                                      l.offsets = append(l.offsets, l.i)
-                                      l.mend = l.i
-
-<lstateName>([ \t]*)\{                l.state = lstateLabels
-
-<lstateName>[ \t]+                    l.state = lstateValue
-                                      l.vstart = l.i
-
-
-<lstateLabels>[ \t]+
-<lstateLabels>,?\}                    l.state = lstateValue
-                                      l.mend = l.i
-<lstateLabels>(,?[ \t]*)              l.state = lstateLName
-                                      l.offsets = append(l.offsets, l.i)
-
-<lstateLName>{L}({L}|{D})*            l.state = lstateLEq
-                                      l.offsets = append(l.offsets, l.i)
-
-<lstateLEq>[ \t]*=                    l.state = lstateLValue
-
-<lstateLValue>[ \t]+
-<lstateLValue>\"                      l.state = lstateLValueIn
-                                      l.offsets = append(l.offsets, l.i)
-<lstateLValueIn>(\\.|[^\\"])*\"       l.state = lstateLabels
-                                      if !utf8.Valid(l.b[l.offsets[len(l.offsets)-1]:l.i-1]) {
-                                        l.err = fmt.Errorf("invalid UTF-8 label value")
-                                        return -1
-                                      }
-                                      l.offsets = append(l.offsets, l.i-1)
-
-<lstateValue>[ \t]+                   l.vstart = l.i
-<lstateValue>(NaN)                    l.val = math.Float64frombits(value.NormalNaN)
-                                      l.state = lstateTimestamp
-
-<lstateValue>[^\n \t\r]+              // We don't parse strictly correct floats as the conversion
-                                      // repeats the effort anyway.
-                                      l.val, l.err = strconv.ParseFloat(yoloString(l.b[l.vstart:l.i]), 64)
-                                      if l.err != nil {
-                                          return -1
-                                      }
-                                      l.state = lstateTimestamp
-
-<lstateTimestamp>[ \t]+               l.tstart = l.i
-<lstateTimestamp>{D}+                 ts, err := strconv.ParseInt(yoloString(l.b[l.tstart:l.i]), 10, 64)
-                                      if err != nil {
-                                          l.err = err
-                                          return -1
-                                      }
-                                      l.ts = &ts
-<lstateTimestamp>[\r\n]+              l.nextMstart = l.i
-                                      return 1
-<lstateTimestamp>\0                   return 1
+\0                                    return tEOF
+\r?\n                                 l.state = sInit; return tLinebreak
```

Comment:
\n is the only valid linebreak

Reply:
I can get rid of that but this is not something I added in this PR. So we might see people file issues after changing it. Though unlikely probably.

Reply:
It is a bug, and you may aswell fix it while you're chaning all of this code.